### PR TITLE
Refactored model train and test part

### PR DESCRIPTION
**Pull request by hovogrig**

### CODE REVIEW

File: server/controllers.py [hunk 1]
```diff
@@ -60,4 +59,18 @@ def get_emotion_by_id(emotion_id : int):
     return {"emotion": emotions[emotion_id]} 
 
 if __name__ == "__main__":
+    
+    model = CNN()
```
Comment (melllinia) [inline server/controllers.py]: Initialize the model outside of the main function, to make it accessible for APIs, or make it a global variable.

File: server/controllers.py [hunk 2]
```diff
@@ -60,4 +59,18 @@ def get_emotion_by_id(emotion_id : int):
...
+
+    # Load a model
+    checkpoint = torch.load('/home/hovhannes/Desktop/FacialExpressionRecognition/model/checkpoints/model.pkl')
+    model.load_state_dict(checkpoint['model_state_dict'])
+    optimizer.load_state_dict(checkpoint['optimizer_state_dict'])
+    epoch = checkpoint['epochs']
+
+    # Save this in train stage
+    # accuracy_on_validation_set = evaluate(model, val_loader)
+    # accuracy_on_train_set = evaluate(model, train_loader)
```
Comment (melllinia) [inline server/controllers.py]: Please confirm: to make a prediction you always need a DataLoader?

Comment (hovogrig) [reply server/controllers.py]: For making a prediction, we need to image pass through our transform. After that we can give it to our model, without any DataLoader.

File: server/controllers.py (hunk 1 above)
Comment (hovogrig) [reply server/controllers.py]: Moved to global scope, thanks.

**Review by melllinia — APPROVED** [review]
Looks good, confirmed!

**Pull request merged**